Source/WebCore/html/HTMLMediaElement.h

```diff
@@ -1225,6 +1229,9 @@ class HTMLMediaElement
 
     VideoFullscreenMode m_videoFullscreenMode { VideoFullscreenModeNone };
     bool m_videoFullscreenStandby { false };
+#if ENABLE(FULLSCREEN_API)
+    bool m_isChildOfDocumentFullscreenElement { false };
```

Comment:
```suggestion
    bool m_isChildOfElementFullscreen { false };
```

to be in line with the nomenclature used in the file and typically when referred to it..